doc/Type/independent-routines.pod6

```diff
@@ -688,13 +722,32 @@ Between the C<%> and the format letter, you may specify several
 additional attributes controlling the interpretation of the format. In
 order, these are:
 
-=head3 Format parameter index
+=head3 Format parameter index using the '$' symbol
 
-An explicit format parameter index, such as C<2$>. By default,
-C<sprintf> will format the next unused argument in the list, but this
-allows you to take the arguments out of order:
+An explicit format parameter integer index (ranging from 1 to N args)
```

Comment:
I can't seem to parse this sentence. Do you mean it's an integer index *for* the format parameter?

Reply:
"integer" is reundant here, yes.

Reply:
Well, an index is always an integer, that's true. But I guess the intention was to express that it was an integer, as in a non-zero UInt. Probably it would be clearer to just say it starts at 1.

Reply:
Will fix.